codegen-hetarchief.yml

```diff
@@ -1,6 +1,6 @@
 overwrite: true
 schema:
-  - ${GRAPHQL_URL_HET_ARCHIEF}:
+  - http://localhost:9000/v1/graphql:
```

Comment:
should this be committed? maybe its better to default it to localhost:9000 indeed, but then i would add the old line in a comment, so we can still pull types from the tst database if bart changes something and we don't have the hetarchief hasura running locally

Reply:
I was a bit to eager to add stuff to the commit message. This should indeed not be committed.